pkg/chartmuseum/server/multitenant/cache.go

```diff
@@ -472,16 +472,17 @@ func (server *MultiTenantServer) newRepositoryIndex(log cm_logger.LoggingFn, rep
 }
 
 func (server *MultiTenantServer) initCacheTimer() {
-	if server.CacheInterval > 0 {
-		// delta update the cache every X duration
-		// (in case the files on the disk are manually manipulated)
-		go func() {
-			t := time.NewTicker(server.CacheInterval)
-			for _ = range t.C {
-				server.rebuildIndex()
-			}
-		}()
-	}
+	if server.CacheInterval == 0 {
+		server.CacheInterval = defaultCacheInterval
```

Comment:
could this be set as the default [here](https://github.com/helm/chartmuseum/blob/main/pkg/config/vars.go#L777) instead?